### ISSUE TITLE
False positive: Indentation on file with tabs

### ISSUE THREAD

**Issue by lrozenblyum** [checkstyle/checkstyle#20339]
I have read check documentation: https://checkstyle.org/checks/misc/indentation.html#Indentation
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

Attention! The file contains tab characters!

```cmd
e:\temp\CheckstyleReport>type config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <property name="tabWidth" value="4"/>
    <module name="TreeWalker">
      <module name="Indentation" />
    </module>
</module>

e:\temp\CheckstyleReport>type Test.java
public class Test {
        public static void main(String[] args) {
                throw new RuntimeException(
                        "This is a test exception"
                );
        }
}

e:\temp\CheckstyleReport>set RUN_LOCALE="-Duser.language=en -Duser.country=US"

e:\temp\CheckstyleReport>java %RUN_LOCALE% -jar checkstyle-13.5.0-all.jar -c config.xml Test.java
Starting audit...
[ERROR] e:\temp\CheckstyleReport\Test.java:4:13: 'new' child has incorrect indentation level 12, expected level should be 16. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```

**Describe what you expect in detail.**
The indentation should be treated as fine.

**Additional details**
It's a regression, it was fine in 13.0.0 and is not ok in 13.5.0.
